### PR TITLE
Use mysql.escape/escapeId()

### PR DESCRIPTION
**Pull request by raymondfeng**
/to @bajtos or @ritch

### CODE REVIEW

File: lib/mysql.js [hunk 1]
```diff
@@ -871,7 +873,8 @@ MySQL.prototype.alterTable = function (model, actualFields, actualIndexes, done,
   function actualize(propName, oldSettings) {
     var newSettings = m.properties[propName];
     if (newSettings && changed(newSettings, oldSettings)) {
-      sql.push('CHANGE COLUMN `' + propName + '` `' + propName + '` ' +
+      var pName = self.client.escapeId(propName);
+      sql.push('CHANGE COLUMN ' + pName + ' ' + pName + ' ' +
```
Comment (ritch) [inline lib/mysql.js]: Was this change intentional?

Comment (raymondfeng) [inline lib/mysql.js]: Yes. Why do you ask?

**Comment by ritch** [comment]
The changes look good. I am not deeply familiar with this code so I'm not sure if you've missed any strings.

File: lib/mysql.js (hunk 1 above)
Comment (ritch) [inline lib/mysql.js]: Looks like the arg was propName. Just wanted to confirm.
